README.md

```diff
@@ -8,7 +8,7 @@ Compose Switch is a replacement to the Compose V1 `docker-compose` (python) exec
 We provide an script for automated installation:
 
 ```console
-$ curl -fL https://raw.githubusercontent.com/docker/compose-cli/main/scripts/install/install_linux.sh | sh
+$ curl -fL https://raw.githubusercontent.com/docker/compose-switch/master/install_on_linux.sh | sh
```

Comment:
LGTM but we could also add the missing `sudo` as you've stated in https://github.com/docker/compose-switch/issues/6

```suggestion
$ curl -fL https://raw.githubusercontent.com/docker/compose-switch/master/install_on_linux.sh | sudo bash -
```